Main/ClientApp/src/Components/Dialogs/SimpleDialog.tsx

```diff
@@ -21,6 +21,7 @@ type SimpleDialogProps = {
   description?: string;
   content?: ReactNode;
   actions?: ReactNode;
+  additionalCancelAction?: () => void;
```

Comment:
Is it additional if there is no "cancelAction"? I would rename those into just cancelAction and confirmAction and pass the actions together

Reply:
cancel action is always in SimpleDialog and provides closing the dialog, but if it is needed it can also do additional action.  innormal case when you implement new dialog which uses SimpleDialog you don't bother in cancel action